validator/sawtooth_validator/consensus/poet1/wait_certificate.py

```diff
@@ -75,11 +75,16 @@ def create_wait_certificate(cls,
             journal.consensus.poet1.wait_certificate.WaitCertificate: A new
                 wait certificate.
         """
-
-        enclave_certificate = \
-            cls.poet_enclave.create_wait_certificate(
-                wait_timer=wait_timer,
-                block_digest=block_digest)
+        enclave_certificate = None
+        try:
+            enclave_certificate = \
+                cls.poet_enclave.create_wait_certificate(
+                    wait_timer.enclave_wait_timer,
+                    block_hash)
+        except Exception as ex:
```

Comment:
Yeah....it somehow got past the lint filters in the 0-7 branch, but got caught in master.  I need to "backport" tha fix.